.github/workflows/linux.bash

```diff
@@ -1,12 +1,40 @@
 #!/bin/bash
 set -exo pipefail
 
-export PATH="/usr/lib/ccache:/usr/lib64/ccache:$PATH"
+export PATH="/usr/lib/ccache/bin:/usr/lib/ccache:/usr/lib64/ccache:$PATH"
 export CCACHE_DIR=/icinga2/ccache
 export CTEST_OUTPUT_ON_FAILURE=1
 CMAKE_OPTS=''
 
 case "$DISTRO" in
+  alpine:*)
+    # Install packages from Alpine package, just
+    # - LibreSSL instead of OpenSSL 3 and
+    # - no MariaDB or libpq as they depend on OpenSSL.
+    # https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/icinga2/APKBUILD
+    apk add \
+      bison \
+      boost-dev \
+      ccache \
+      cmake \
+      flex \
+      g++ \
+      libedit-dev \
+      libressl-dev \
+      ninja-build \
+      samurai \
+      tzdata \
+      yajl-dev
```

Comment:
No one in this file:

Absolutely no one in this file:

@oxzi:

Let's put each package on one line. 😈

Reply:
This was intentional, yes. First, doing so allowed me sorting the packages alphabetically. But mostly, this eases the readability of future diffs as they only contain the packages in question.

If putting all packages in one line, while honoring a line width of 80, this would like the following mess:
```
    apk add bison boost-dev ccache cmake flex g++ libedit-dev libressl-dev \
      ninja-build samurai tzdata yajl-dev
```

Reply:
1. Re-consider samurai (https://github.com/Icinga/icinga2/pull/9949#discussion_r1908996845)
2. Re-consider yajl-dev (https://github.com/Icinga/icinga2/pull/9949#discussion_r1908997658)
3. Consider this simple trick: `{a,b,c}-d` (https://github.com/Icinga/icinga2/blob/v2.14.3/.github/workflows/linux.bash#L13)
4. The packages list got smaller 👍

Reply:
My aim was readability, not code golfing. I squeezed the packages into one line.